pkg/migration/plan_executor.go

```diff
@@ -26,9 +26,11 @@ const (
 // PlanExecutor drives the execution of a plan's steps and
 // uses the configured `executors` to execute those steps.
 type PlanExecutor struct {
-	executors []Executor
-	plan      Plan
-	callback  ExecutorCallback
+	executors          []Executor
+	plan               Plan
+	callback           ExecutorCallback
+	LastSuccessfulStep int
```

Comment:
Where do we read `LastSuccessfulStep`?

Reply:
We read this value in the family-migrator. Please see this [PR](https://github.com/upbound/extensions-migration/pull/18).